README.md

```diff
@@ -41,11 +41,16 @@
   </h3>
 </div>
 
-A modular web framework built around async/await
+Tide is a minimal and pragmatic Rust web application framework built for
+rapid development. It comes with a robust set of features that make building
+database-backed web applications easier and more fun.
```

Comment:
Database-backed seems more specific/constrained than tide is? Suggestion: "that make building web applications and apis easier and more fun."

Reply:
I think it may still be worth mentioning futures/async in the description as well.